test/abstracts.md

```diff
@@ -8,6 +8,39 @@ It's rather a specification than current implementation,
 but the goal is to optimize such specification as much as possible
 and achieve best performance on top of that.
 
+
+
+### Definitions
+
+`hyperHTML` is a function tag for template literals.
+
+Its `statics` argument is a unique templateObject
+http://www.ecma-international.org/ecma-262/6.0/#sec-gettemplateobject
+
+Its `context` MUST be a DOM Element.
+
+`hyperHTML.bind(context)` creates a new tag for such `context`.
```

Comment:
Note: ... couple of things. Maybe avoid the word "tag" as it can be confused with "HTML tag". 

Also, "creates a new tag for such `context`." is not strictly accurate: at this point it just does ES `bind()`. 

It only gets interesting when the function tag is called:

```
const render = hyperHTML.bind(context);
render`
   <!-- this represents the contents of the DOM node -->
`;
```  

(I know you know this, but I'm imagining someone who doesn't understand the innards 😊)